cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Insert.hs

```diff
@@ -196,11 +196,13 @@ insertShelleyBlock syncEnv shouldLog withinTwoMins withinHalfHour blk details is
 
     insertStakeSlice syncEnv $ apStakeSlice applyResult
 
+    when (ioGov iopts)
+      . lift
+      $ insertOffChainVoteResults tracer (envOffChainVoteResultQueue syncEnv)
+
     when (ioOffChainPoolData iopts && unBlockNo (Generic.blkBlockNo blk) `mod` offChainModBase == 0)
```

Comment:
Since this no longer does queries, we could call it for every block, so the `offChainModBase` check in unecessary.